src/theme/components/OpenPositions.js

```diff
@@ -1,33 +1,41 @@
 import React from "react";
 import Link from "@docusaurus/Link";
 
+import axios from "axios";
 import clsx from "clsx";
+
 import styles from "./openPositions.module.css";
 
-import { useHistory } from "@docusaurus/router";
+const LEVER_ORGANIZATION = 'ironfish';
 
 function Position({ title, time, location, link }) {
-  const history = useHistory();
-
-  function handleClick() {
-    window.open(link, "_blank");
-  }
-
   return (
-    <div className={clsx(styles.position)} onClick={handleClick}>
-      <p className={clsx(styles.title)}>
-        <Link className={clsx(styles.button)} to={link}>
-          {title}
-        </Link>
-      </p>
-      <p className={clsx(styles.time)}>{time}</p>
-      <p className={clsx(styles.location)}>{location}</p>
-      <img src="/img/careers/arrow.svg" alt="link" />
-    </div>
+    <Link className={clsx(styles.positionLink)} to={link}>
+      <div className={clsx(styles.position)}>
+        <p className={clsx(styles.title)}>{title}</p>
+        <p className={clsx(styles.time)}>{time}</p>
+        <p className={clsx(styles.location)}>{location}</p>
+        <img src="/img/careers/arrow.svg" alt="link" />
+      </div>
+    </Link>
   );
 }
 
 function OpenPositions() {
+  const [jobData, setJobData] = React.useState({ type: 'loading' })
+
+  React.useEffect(() => {
+    const fetchJobs = async () => {
+      try {
+        const result = await axios.get(`https://api.lever.co/v0/postings/${LEVER_ORGANIZATION}?mode=json`)
```

Comment:
This isn't a request for change, only a note.

I noticed that a lot of our front end is written this way. It looks this re-create hooks like useAsync and this suffers from unmounted component errors.